### ISSUE TITLE
Statement parsing issue

### ISSUE THREAD

**Issue by feraudet** [flosell/iam-policy-json-to-terraform#10]
Hello,
This valid policy : 
```{
  "Version" : "2012-10-17",
  "Statement" : {
    "Effect" : "Deny",
    "Action" : "*",
    "Resource" : "*",
    "Condition" : {
      "NotIpAddress" : {
        "aws:SourceIp" : [
          "xxx.xxx.xxx.xxx/xx",
          "yyy.yyy.yyy.yyy"
        ]
      },
      "StringNotLike" : {
        "aws:invokedBy" : "*.amazonaws.com"
      }
    }
  }
}
```
make iam-policy-json-to-terraform return : 
`unable to convert: json: cannot unmarshal object into Go struct field jsonPolicyDocument.Statement of type []converter.jsonStatement`

I had to add square brackets to make it working

Thanks for this very useful tool

**Comment by nitrocode** [comment]
The converted json should render this

```hcl
provider "aws" {
  region = "us-west-2"
}

data "aws_iam_policy_document" "this" {
  statement {
    effect    = "Deny"
    actions   = ["*"]
    resources = ["*"]

    condition {
      test     = "NotIpAddress"
      variable = "aws:SourceIp"

      values = [
        "xxx.xxx.xxx.xxx/xx",
        "yyy.yyy.yyy.yyy"
      ]
    }

    condition {
      test     = "StringNotLike"
      variable = "aws:invokedBy"

      values = [
        "*.amazonaws.com"
      ]
    }
  }
}

output "policy_json" {
  value = data.aws_iam_policy_document.this.json
}
```

I wonder what other `statement` attributes are also not configured. The full list of attributes are on the [data source aws_iam_policy_document](https://www.terraform.io/docs/providers/aws/d/iam_policy_document.html). I also wonder if the aws provider has a similar json structure that we can reuse instead of building on the [bespoke one in this project](https://github.com/flosell/iam-policy-json-to-terraform/blob/master/converter/decode.go#L13).

**Comment by flosell** [comment]
Hi @feraudet, thanks for reporting this! 

Just to clarify my understanding, you have a policy with a single statement, that causes an error when expressed as an object like so:

```
{
  "Version" : "2012-10-17",
  "Statement" : {
    ... 
  }
}
```

However it everything works fine when wrapping the statement in an array like this: 

```
{
  "Version" : "2012-10-17",
  "Statement" : [{
    ... 
  }]
}
```

I was able to reproduce this and I'm looking into how to fix it in an elegant way (AWS' philosophy of collapsing single-item arrays clashes with Golangs JSON Parser that expects a single, predictable schema...). 

@nitrocode I don't think this is related to missing statement attributes, at least I couldn't find a missing attribute when comparing with the terraform docs. Did I miss one?

**Comment by nitrocode** [comment]
Ah I was mistaken. All the keys are present. I was also considering in the future if AWS added anything to the struct that it may be better to use the one in the provider but who knows when they will modify IAM policy structure again.

I tried it with the square brackets and this tool converted the JSON into HCL as expected. I also confirmed that the JSON policy will work without the square brackets upon `terraform apply`.

```hcl
resource "aws_iam_policy" "policy" {
  name        = "test_policy"
  path        = "/"
  description = "My test policy"

  policy = <<EOF
{
  "Version" : "2012-10-17",
  "Statement" : {
  }
}
EOF
}
```

Would it be possible to update the code to allow it to work for input JSON with a statement key/value in addition to the standard statement array?
